src/main/java/uk/gov/companieshouse/pscfiling/api/service/impl/PscDetailsServiceImpl.java

```diff
@@ -16,6 +16,9 @@
 import uk.gov.companieshouse.pscfiling.api.service.PscDetailsService;
 import uk.gov.companieshouse.pscfiling.api.utils.LogHelper;
 
+/**
+ * Interacts with the external CHS PSC API service to retrieve PSCs.
+ */
```

Comment:
this doco is in the interface, so not needed here as well

Reply:
Updated.